README.md

```diff
@@ -8,6 +8,12 @@ This repo contains Ansible code for OpenShift and Atomic Enterprise.
   ```
     dnf install -y ansible-1.9.4 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
   ```
+  - RHEL 7:
+  ```
+  yum install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  rpm --import http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+  yum install ansible1.9-1.9.4-2.el7 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
```

Comment:
Running the `yum install ansible-1.9.4` does not end with success on RHEL 7.
(and, openshift-ansible is not compatible with ansible 2.0 ).
The only option I could find to install 1.9.4 on RHEL is adding the epel repo, so I can install: ansible1.9-1.9.4-2.el7.